templates/pages/brand.html

```diff
@@ -17,7 +17,10 @@
 {{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
 {{#if brand.image}}
     <div class="brand-image-container">
-        <img class="lazyload" data-sizes="auto" src="{{cdn 'img/loading.svg'}}" data-src="{{getImage brand.image 'thumb_size'}}">
+        {{> components/common/responsive-img
+            image=brand.image
+            lazyload='disabled'
```

Comment:
Brand image is critical above-the-fold content on the brand page, so not lazyloading this.